src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -16,12 +19,19 @@ public class FindCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all internships whose company names contain any"
             + " of the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " apple google facebook";
-
-    private final CompanyNameContainsKeywordsPredicate predicate;
-
-    public FindCommand(CompanyNameContainsKeywordsPredicate predicate) {
+            + "Parameters: "
+            + "[" + PREFIX_COMPANY_NAME + "NAME_KEYWORD]"
+            + "[" + PREFIX_STATUS + "STATUS_KEYWORD]"
+            + "[" + PREFIX_TAG + "TAG_KEYWORD]...\n"
+            + "Example: " + COMMAND_WORD
+            + PREFIX_COMPANY_NAME + "apple google facebook"
+            + PREFIX_STATUS + "applied interview offered"
+            + PREFIX_TAG + "python"
+            + PREFIX_TAG + "java";
```

Comment:
Currently, this prints out the following in the UI:

![image](https://user-images.githubusercontent.com/86542359/223998263-09d102bc-6e3e-4ddf-a60c-ec338d7fd276.png)

Would it be better if you could rephrase the instruction to reflect what the `updated` find command can do, where it is more than just finding matches in `CompanyName`. Also, there needs to be spacing between the `t/` tags in the example given.